tests/utils/editor_recommendation_test.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+
+from data_science_pipeline.utils.editor_recommendation import (
+    get_author_ids_of_given_version_of_paper
+)
+
+MANUSCRIPT_VERSION_1 = "match_version_id_1"
```

Comment:
would call this `MANUSCRIPT_VERSION_ID_1`.
otherwise it could be mistaken for an object representing a manuscript version.